pkg/operator/controller/ingress/status_test.go

```diff
@@ -108,6 +111,131 @@ func cond(t string, status operatorv1.ConditionStatus, reason string, lt time.Ti
 	}
 }
 
+func TestComputeIngressDegradedCondition(t *testing.T) {
+	tests := []struct {
+		name           string
+		conditions     []operatorv1.OperatorCondition
+		expectedStatus operatorv1.ConditionStatus
```

Comment:
Does this read a little better if it was `degradedStatus`. Might just help clarify what we're asserting.

Reply:
I have changed it to `expectIngressDegradedStatus` (and in `TestComputeDeploymentDegradedCondition`, I have changed it to `expectDeploymentDegradedStatus`).